src/aiidalab_qe/plugins/pdos/setting.py

```diff
@@ -103,12 +114,14 @@ def render(self):
             """),
             ipw.HTML("""
                 <div style="line-height: 140%; padding-top: 0px; padding-bottom: 5px">
-                    By default, the tetrahedron method is used for PDOS calculation.
-                    If required you can apply Gaussian broadening with a custom degauss
-                    value.
+                    By default, the <b>tetrahedron method</b> is used for partial density of states (PDOS) calculation.
+                    However, if you need more control over the broadening, you can apply <b>Gaussian broadening</b>
+                    by specifying a custom <b>degauss</b> value.
                     <br>
-                    For molecules and systems with localized orbitals, it is
-                    recommended to use a custom degauss value.
+                    <b>Recommendation for Molecules and Localized Orbitals:</b><br>
+                    For systems involving molecules or localized orbitals, it is recommended to use a
```

Comment:
```suggestion
                    <b>Recommendation for Molecules and Localized Orbitals:</b>
                    <br>
                    For systems involving molecules or localized orbitals, it is recommended to use a
```
I find it a bit clearer to give `<br>` tags their own line 🙂